libethereum/Executive.cpp

```diff
@@ -229,7 +229,9 @@ bool Executive::execute()
 
 bool Executive::call(Address _receiveAddress, Address _senderAddress, u256 _value, u256 _gasPrice, bytesConstRef _data, u256 _gas)
 {
-	CallParameters params{_senderAddress, _receiveAddress, _receiveAddress, _gas, _value, _data, {}, {}};
+	dev::bytes defaultOutput = dev::fromHex("0000000000000000000000000000000000000000000000000000000000000000");
+	dev::bytesRef _out(&defaultOutput.at(0), 32);
```

Comment:
Would &bytes.at(0) return the address of a temporarily created reference rather than the address of bytes?
Try dev::bytesRef _out(defaultOutput, 32);